planetarium/NineChronicles
# Implement `Create Spine Prefab (All NPC)` menu.

### Description

SSIA

### How to test

1. Check if `Tools/9C/Create Spine Prefab(All NPC)` menu works properly in unity editor.

### Related Links
https://app.asana.com/0/958521740385861/1201200786778975

### Required Reviewers
@planetarium/9c-dev
